src/index.ts

```diff
@@ -1,11 +1,11 @@
 import { Lottery } from "./lottery-application";
-import * as fs from "fs";
-import {writeFile} from "fs";
+import {FileExplorer} from "./components/FileExplorer";
 
-const upperBoundary = 200_000;
-const numbersToDraw = 199_900;
+const upperBoundary = 2000;
+const numbersToDraw = 1999;
```

Comment:
Modified because these values take up 100 percent of CPU usage and high memory utilization. Difficult to test.

Reply:
Yes, please find a way of passing these params via command line arguments. I don't exactly know how to do this in node.
But, the idea of this exercise is to stress the algorithm. With a small amount of numbers to draw, we cannot see the differences in the algorithms (more on that later, new task coming).